runners/wikiConsole/templates/downloadFile.asciidoc

```diff
@@ -0,0 +1,22 @@
+Download a File
+===============
+
+Download
+~~~~~~~~
+First, you need to open a command prompt in your current Workspace. For Windows you can use Powershell and on Linux you can use the Terminal.
+
+<%if(!directoryExists){ %>
+You can specify a folder for your download, but this folder should be existing. We create a folder with this command where we can load the files into.
+`mkdir <%=dir;%>`
+<% } %>
+
+Now we can start download the file with the wget command. You can read some Information about the usage on Linux https://manpages.ubuntu.com/manpages/hirsute/en/man1/wget.1.html[here].
+
+On Windows wget command is an alias for Invoke-WebRequest and mostly works like wget on Linux, but a detailed documentation can be found https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-7.1[here].
```

Comment:
I would find it better to use the Invoke-WebRequest command here, because I think this is the more natural way when working with Powershell. But you can add the information that wget is an alias and is also possible

Reply:
I changed it